src/rgw/rgw_op.cc

```diff
@@ -947,6 +950,8 @@ int RGWGetObj::verify_permission(optional_yield y)
   }
 
   if (!verify_object_permission(this, s, action)) {
+    verify_perimission_span->set_tag("type","error");
```

Comment:
we should be consistent with the strings we use. e.g. do we use "msg" or "log" as the key for the log values?
also with the tag names that repeat in multiple places, we should use some const strings to avoid differences and spelling mistakes.
e.g. if someone wants to run a query on a bucket across different operations and traces, they should use the same tag name.
also, lets be consistent with the log values describing similar issues - e.g. "permission denied"

Reply:
if we want to be consistent with the key of the log values, should we just "hide" that param, so `set_log` would receive only the log value?